js/committee.js

```diff
@@ -28,117 +28,170 @@ function highlightSelection(turnOn)
 
-    // highlight embedded tab 
+    // highlight embedded-development tab 
     highlightSelection("embedded");
 }
 
 
-// Cybersecurity Data is initialized with this function 
+
+/*          Cybersecurity Body Content function              */ 
 function showCyber()
 {
-    // Select content title and body elements. 
+    // select content title and body elements
     let title = document.getElementById('content-title'); 
     let body = document.getElementById('content-body'); 
 
-    // Set Content title. 
+    // set content title  
     title.innerText = 'Cybersecurity Committee'; 
 
+<<<<<<< Updated upstream
     // Create some markup to write to page displaying an image and text via query select
-    let codeline = '<img class="content" src="images/cyber_img.png"><br><br><div>' + $("#info-cybersecurity").text() + '</div><br>';
+    let codeline = '<img class="content" src="images/cyber_img.png"><br><br><div>' + $("#info-cybersecurity").text() + '</div>';
+=======
+    // create image element 
+    const img = document.createElement('img');
+    img.classList.add('content');
+    img.src = 'images/cyber_img.png';
 
-    // Write markup to the document. 
-    body.innerHTML = codeline; 
+    // create description element
+    const description = document.createElement('div');
+    description.innerHTML = $("#info-cybersecurity").text();
+>>>>>>> Stashed changes
 
-    // highlight cyber tab
+    // add elements into content node 
+    body.replaceChildren(img, document.createElement('br'), document.createElement('br'), description);
+
+    // highlight cybersecurity tab
     highlightSelection('cyber');
 }
 
 
-// Girls who code data initialization function 
+
+/*          Girls-Who-Code Body Content function              */
 function showGWC()
 {
-    // query select by id the right column title and description 
+    // select title and body elements
     let title = document.getElementById('content-title'); 
     let body = document.getElementById('content-body'); 
 
-    // Set the right div title. 
+    // set content title 
     title.innerText = 'Girls Who Code Committee'; 
 
+<<<<<<< Updated upstream
     // Create some markup to page displaying the committee image and description 
     let codeline = '<img class="content" src="images/gwc_logo1.png"><br><br><div>' + 
-    $("#info-girlswhocode").text() + '<br>Check out their website at:  <a style="color:white; font-size:bolder;" href="https://girlswhocode.com/" target="_blank">www.girlswhocode.com</a></div><br>'; 
+    $("#info-girlswhocode").text() + '<br>Check out their website at:  <a style="color:white; font-size:bolder;" href="https://girlswhocode.com/" target="_blank">www.girlswhocode.com</a></div>'; 
+=======
+    // create image element 
+    const img = document.createElement('img');
+    img.classList.add('content');
+    img.src = 'images/gwc_logo1.png';
+>>>>>>> Stashed changes
```

Comment:
There are several of these commit headers throughout the file, could you clean it up by removing them? Also check out a887448462535155059f9cc1787464c5c3825dd5 for some jQuery changes I made and merge them into this branch.
Otherwise, looks good!